### PR TITLE
fix(DB/Locale): ruRU Client Locale

### PR DESCRIPTION
**Pull request by SergeyBliznec**
Adding Russian language support for most gameplay elements (quests, books, game objects, etc.).

## Changes Proposed:
This PR proposes changes to:
Adds missing Russian strings in localization tables.

## Issues Addressed:

- Closes 
https://github.com/azerothcore/azerothcore-wotlk/issues/15135
https://github.com/azerothcore/azerothcore-wotlk/issues/14222
https://github.com/azerothcore/azerothcore-wotlk/issues/14228
https://github.com/azerothcore/azerothcore-wotlk/issues/14225
https://github.com/azerothcore/azerothcore-wotlk/issues/14224
https://github.com/azerothcore/azerothcore-wotlk/issues/14208
https://github.com/azerothcore/azerothcore-wotlk/issues/14229
https://github.com/azerothcore/azerothcore-wotlk/issues/14226
https://github.com/azerothcore/azerothcore-wotlk/issues/14208

## SOURCE:

https://github.com/azerothcore/azerothcore-wotlk/issues?q=russian
https://github.com/SergeyBliznec/azerothcore-rudb
https://github.com/NeatElves/wotlk-ytdbdb

## Tests Performed:

Tested on LAN server. No problems found.
Before:

After:

Before:

After:

## How to Test the Changes:

1. Requires ruRU client

## Known Issues and TODO List:

The quest_getering_locale table doesn't work.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by VitaliyBabkov — APPROVED** [review]
I confirm that the quests have been translated.
Снимок экрана 2023-09-06 120226
Снимок экрана 2023-09-06 120245
Снимок экрана 2023-09-06 120302
Снимок экрана 2023-09-06 120317
Снимок экрана 2023-09-06 120333
Снимок экрана 2023-09-06 120349
Снимок экрана 2023-09-06 120421

**Review by VitaliyBabkov — APPROVED** [review]
Game objects also received a translation.

File: data/sql/updates/pending_db_world/achievement_reward_locale.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+SET time_zone = "+00:00";
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+
+
+REPLACE INTO `achievement_reward_locale` (`ID`, `Locale`, `Subject`, `Text`) VALUES
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/achievement_reward_locale.sql:12]: no. dont use replace into. use delete insert or update

see https://www.azerothcore.org/wiki/sql-standards

**Review by Kitzunu — CHANGES_REQUESTED** [review]
Also just do it for the texts you are actually fixing. Do not just re-add every text that already exists.

**Comment by Kitzunu** [comment]
I will close this as it is not an acceptable format for the SQL changes.

**Pull request closed without merging**